### PR TITLE
Update eg.m3u

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by BellezaEmporium — COMMENTED** [review]
I don't need to have the user-agent on my side. Works perfectly using VLC.

**Comment by Sphinxroot** [comment]
vlc it work and kodi need the user agent  .. also please correct logo .. they are in  group-title=

**Review by Sphinxroot — DISMISSED** [review]
logo
#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="EG" tvg-language="Arabic" tvg-logo="" group-title="http://www.niletc.tv/images/cinema/cinema.png",Nile TV Cinema

#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="EG" tvg-language="Arabic" tvg-logo="http://www.niletc.tv/images/cinema/cinema.png" group-title="",Nile TV Cinema

File: channels/eg.m3u [hunk 1]
```diff
@@ -19,24 +17,35 @@ https://linkastream.co/headless?url=https://www.youtube.com/c/CBCSofraStream/liv
...
 https://mn-nl.mncdn.com/mekameleen/smil:mekameleentv.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="Mekameleenmkmlyn.eg" tvg-name="Mekameleen مكملين" tvg-country="EG" tvg-language="" tvg-logo="" group-title="",Mekameleen مكملين (1080p)
-http://mn-nl.mncdn.com/mekameleen/smil:mekameleentv.smil/chunklist_b1864000.m3u8
 #EXTINF:-1 tvg-id="MESat.eg" tvg-name="MESat" tvg-country="EG" tvg-language="Arabic" tvg-logo="https://copticocc.org/mesat/wp-content/uploads/2018/07/logo-width12.png" group-title="Religious",MESat [Not 24/7]
 https://linkastream.co/headless?url=https://www.youtube.com/channel/UCg5uHOxrP5GkMWldOavPKGQ/live
-#EXTINF:-1 tvg-id="MokameleenTV.eg" tvg-name="Mokameleen TV" tvg-country="EG" tvg-language="" tvg-logo="" group-title="",Mokameleen TV (1080p)
-http://mn-nl.mncdn.com/mekameleen/smil:mekameleentv.smil/chunklist_b1864000.m3u8?v=787761179
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="EG" tvg-language="Arabic" tvg-logo="" group-title="http://www.niletc.tv/images/cinema/cinema.png",Nile TV Cinema
```
Comment (LaneSh4d0w) [inline channels/eg.m3u:28]:
```suggestion
#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="EG" tvg-language="Arabic" tvg-logo="http://www.niletc.tv/images/cinema/cinema.png" group-title="",Nile TV Cinema
```

File: channels/eg.m3u [hunk 2]
```diff
@@ -19,24 +17,35 @@ https://linkastream.co/headless?url=https://www.youtube.com/c/CBCSofraStream/liv
...
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.124 Safari/537.36
+https://livestreaming5.onlinehorizons.net/hls-live/NTNNileCinema/_definst_/liveevent/livestream.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="EG" tvg-language="Arabic" tvg-logo="" group-title="http://www.niletc.tv/images/comedy/comedy.png",Nile TV Comedy
```
Comment (LaneSh4d0w) [inline channels/eg.m3u:31]:
```suggestion
#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="EG" tvg-language="Arabic" tvg-logo="http://www.niletc.tv/images/comedy/comedy.png" group-title="",Nile TV Comedy
```

File: channels/eg.m3u [hunk 3]
```diff
@@ -19,24 +17,35 @@ https://linkastream.co/headless?url=https://www.youtube.com/c/CBCSofraStream/liv
...
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.124 Safari/537.36
+https://livestreaming5.onlinehorizons.net/hls-live/NTNNileComedy/_definst_/liveevent/livestream.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="EG" tvg-language="Arabic" tvg-logo="" group-title="http://www.niletc.tv/images/culture/culture.png",Nile TV Culture
```
Comment (LaneSh4d0w) [inline channels/eg.m3u:34]:
```suggestion
#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="EG" tvg-language="Arabic" tvg-logo="http://www.niletc.tv/images/culture/culture.png" group-title="",Nile TV Culture
```

File: channels/eg.m3u [hunk 4]
```diff
@@ -19,24 +17,35 @@ https://linkastream.co/headless?url=https://www.youtube.com/c/CBCSofraStream/liv
...
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.124 Safari/537.36
+https://livestreaming5.onlinehorizons.net/hls-live/NTNNileCulture/_definst_/liveevent/livestream.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="EG" tvg-language="Arabic" tvg-logo="" group-title="http://www.niletc.tv/images/family/family.png",Nile TV Family
```
Comment (LaneSh4d0w) [inline channels/eg.m3u:37]:
```suggestion
#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="EG" tvg-language="Arabic" tvg-logo="http://www.niletc.tv/images/family/family.png" group-title="",Nile TV Family
```

**Comment by tilfizyawn** [comment]
Thanks everyone for your feedback

**Comment by LaneSh4d0w** [comment]
I could add the different languages for Noble TV, however stream isn't offline for all of them.

**Review by Dum4G — APPROVED** [review]

File: channels/eg.m3u [hunk 5]
```diff
@@ -19,26 +17,37 @@ https://linkastream.co/headless?url=https://www.youtube.com/c/CBCSofraStream/liv
...
+http://livestreaming5.onlinehorizons.net/hls-live/NTNNileFamily/_definst_/liveevent/livestream.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="EG" tvg-language="Arabic" tvg-logo="http://www.niletc.tv/images/learning/learning.png" group-title="Education",Nile TV Learning
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.124 Safari/537.36
+http://livestreaming5.onlinehorizons.net/hls-live/Learning1/_definst_/liveevent/livestream.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="EG" tvg-language="Arabic" tvg-logo="http://www.niletc.tv/images/live/live.png" group-title="",Nile TV Life
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.124 Safari/537.36
+http://livestreaming5.onlinehorizons.net/hls-live/NTNNileLife/_definst_/liveevent/livestream.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="EG" tvg-language="Arabic" tvg-logo="http://www.niletc.tv/images/sport/sport.png" group-title="Sport",Nile TV Sport
```
Comment (Dum4G) [inline channels/eg.m3u:46]:
```suggestion
#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="EG" tvg-language="Arabic" tvg-logo="http://www.niletc.tv/images/sport/sport.png" group-title="Sports",Nile TV Sport
```

File: channels/eg.m3u [hunk 6]
```diff
@@ -19,26 +17,37 @@ https://linkastream.co/headless?url=https://www.youtube.com/c/CBCSofraStream/liv
...
+http://livestreaming5.onlinehorizons.net/hls-live/NTNNileCulture/_definst_/liveevent/livestream.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="EG" tvg-language="Arabic" tvg-logo="http://www.niletc.tv/images/family/family.png" group-title="",Nile TV Family
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.124 Safari/537.36
+http://livestreaming5.onlinehorizons.net/hls-live/NTNNileFamily/_definst_/liveevent/livestream.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="EG" tvg-language="Arabic" tvg-logo="http://www.niletc.tv/images/learning/learning.png" group-title="Education",Nile TV Learning
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.124 Safari/537.36
+http://livestreaming5.onlinehorizons.net/hls-live/Learning1/_definst_/liveevent/livestream.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="EG" tvg-language="Arabic" tvg-logo="http://www.niletc.tv/images/live/live.png" group-title="",Nile TV Life
```
Comment (Dum4G) [inline channels/eg.m3u:43]:
```suggestion
#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="EG" tvg-language="Arabic" tvg-logo="http://www.niletc.tv/images/live/live.png" group-title="Lifestyle",Nile TV Life
```

File: channels/eg.m3u [hunk 7]
```diff
@@ -19,26 +17,37 @@ https://linkastream.co/headless?url=https://www.youtube.com/c/CBCSofraStream/liv
...
+http://livestreaming5.onlinehorizons.net/hls-live/NTNNileCinema/_definst_/liveevent/livestream.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="EG" tvg-language="Arabic" tvg-logo="http://www.niletc.tv/images/comedy/comedy.png" group-title="",Nile TV Comedy
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.124 Safari/537.36
+http://livestreaming5.onlinehorizons.net/hls-live/NTNNileComedy/_definst_/liveevent/livestream.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="EG" tvg-language="Arabic" tvg-logo="http://www.niletc.tv/images/culture/culture.png" group-title="",Nile TV Culture
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.124 Safari/537.36
+http://livestreaming5.onlinehorizons.net/hls-live/NTNNileCulture/_definst_/liveevent/livestream.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="EG" tvg-language="Arabic" tvg-logo="http://www.niletc.tv/images/family/family.png" group-title="",Nile TV Family
```
Comment (Dum4G) [inline channels/eg.m3u:37]:
```suggestion
#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="EG" tvg-language="Arabic" tvg-logo="http://www.niletc.tv/images/family/family.png" group-title="Family",Nile TV Family
```

File: channels/eg.m3u [hunk 8]
```diff
@@ -19,26 +17,37 @@ https://linkastream.co/headless?url=https://www.youtube.com/c/CBCSofraStream/liv
...
 https://mn-nl.mncdn.com/mekameleen/smil:mekameleentv.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="Mekameleenmkmlyn.eg" tvg-name="Mekameleen مكملين" tvg-country="EG" tvg-language="" tvg-logo="" group-title="",Mekameleen مكملين (1080p)
-http://mn-nl.mncdn.com/mekameleen/smil:mekameleentv.smil/chunklist_b1864000.m3u8
 #EXTINF:-1 tvg-id="MESat.eg" tvg-name="MESat" tvg-country="EG" tvg-language="Arabic" tvg-logo="https://copticocc.org/mesat/wp-content/uploads/2018/07/logo-width12.png" group-title="Religious",MESat [Not 24/7]
 https://linkastream.co/headless?url=https://www.youtube.com/channel/UCg5uHOxrP5GkMWldOavPKGQ/live
-#EXTINF:-1 tvg-id="MokameleenTV.eg" tvg-name="Mokameleen TV" tvg-country="EG" tvg-language="" tvg-logo="" group-title="",Mokameleen TV (1080p)
-http://mn-nl.mncdn.com/mekameleen/smil:mekameleentv.smil/chunklist_b1864000.m3u8?v=787761179
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="EG" tvg-language="Arabic" tvg-logo="http://www.niletc.tv/images/cinema/cinema.png" group-title="",Nile TV Cinema
```
Comment (Dum4G) [inline channels/eg.m3u:28]:
```suggestion
#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="EG" tvg-language="Arabic" tvg-logo="http://www.niletc.tv/images/cinema/cinema.png" group-title="Movies",Nile TV Cinema
```

File: channels/eg.m3u [hunk 9]
```diff
@@ -19,26 +17,37 @@ https://linkastream.co/headless?url=https://www.youtube.com/c/CBCSofraStream/liv
...
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.124 Safari/537.36
+http://livestreaming5.onlinehorizons.net/hls-live/NTNNileCinema/_definst_/liveevent/livestream.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="EG" tvg-language="Arabic" tvg-logo="http://www.niletc.tv/images/comedy/comedy.png" group-title="",Nile TV Comedy
```
Comment (Dum4G) [inline channels/eg.m3u:31]:
```suggestion
#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="EG" tvg-language="Arabic" tvg-logo="http://www.niletc.tv/images/comedy/comedy.png" group-title="Comedy",Nile TV Comedy
```

File: channels/eg.m3u [hunk 10]
```diff
@@ -19,26 +17,37 @@ https://linkastream.co/headless?url=https://www.youtube.com/c/CBCSofraStream/liv
...
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.124 Safari/537.36
+http://livestreaming5.onlinehorizons.net/hls-live/NTNNileComedy/_definst_/liveevent/livestream.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="EG" tvg-language="Arabic" tvg-logo="http://www.niletc.tv/images/culture/culture.png" group-title="",Nile TV Culture
```
Comment (Dum4G) [inline channels/eg.m3u:34]:
```suggestion
#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="EG" tvg-language="Arabic" tvg-logo="http://www.niletc.tv/images/culture/culture.png" group-title="Culture",Nile TV Culture
```

**Review by Dum4G — CHANGES_REQUESTED** [review]

**Review by Dum4G — APPROVED** [review]

**Pull request merged**